.buildkite/pipeline.yml

```diff
@@ -0,0 +1,3 @@
+steps:
+  - label: "E2E Test"
```

Comment:
Added as a preparation.

When the catalog-info.yaml file is merged into the main branch, we can modify it to run an actual e2e test using the plugin.